webpack/components/extensions/HostDetails/Tabs/ContentTab/ContentTab.scss

```diff
@@ -0,0 +1,3 @@
+#host-content-tabs {
+  min-height: 2.3rem;
```

Comment:
it works, but I'm against using just a number for this. maybe use a pf variable or wrap the `<Tabs` with `<Section`?